### ISSUE TITLE
As a user, I want to search digitized volumes by title keywords in a clearly marked title field so that I can see what materials are in the archive with a certain title.

### ISSUE THREAD

**Issue by mnaydan** [Princeton-CDH/ppa-django#68]

**Comment by mnaydan** [comment]
Tried varieties of Boolean searches with "art" "poetry" and "English", all working. Used publication filter to filter outside of date range for Bysshe's "art of English poetry," correctly returned 0 works and red box that says "no matching works." Used publication filter to filter from 1750-1800 and query returned 3 results, all within date range. Results do not display thumbnails or text snippets if you are searching just by title. If you search in the title and keyword fields (e.g., keyword "iamb," book title "poetry") it shows thumbnail and snippets. I tried varieties of searches across keyword, author, and title fields and they seem to be working. ONE PROBLEM: I tried doing a title search of "poetry" and filtering by collection (dictionary). I tried clicking on dictionary more than once (there's no visual cue as to which collection has been selected to be filtered) and now I can't get out of the dictionary search filter or deselect it. No matter what I do (try a different search, change pub years) it variably returns 0, 1, or 3 results, all dictionaries.
